dist/tools/pyterm/pyterm

```diff
@@ -303,6 +303,14 @@ class SerCmd(cmd.Cmd):
         date = time.strftime("%Y-%m-%d %H:%M:%S")
         return ["%s" % (date)]
 
+    def do_PYTERM_sleep(self, line):
```

Comment:
Small thing, why not `def do_PYTERM_sleep(self, line=1):`

Reply:
Mostly because I'm clueless about Python, but this one it does not like: `ValueError: could not convert string to float`

I guess these default parameters are not in effect when line is set but is empty.

Reply:
Ok, now the `line` variable name makes sense.  All good with me I guess.